lib/monetary-amount.js

```diff
@@ -8,6 +11,36 @@ const _amount = new WeakMap();
 const _currencyType = new WeakMap();
 const _currencyId = new WeakMap();
 
+function convertAmountOrThrow(amount) {
```

Comment:
The convention for functions that either returns a value or throws is to prefix them with the word "try", not postfix with "OrThrow".

E.g.: `tryConvertAmount`, `tryConvertAddress` etc

Reply:
Refactored to not throw.